command/render.go

```diff
@@ -60,12 +66,19 @@ func (c *RenderCommand) Run(args []string) int {
 
 	args = flags.Args()
 
-	if len(args) != 1 {
+	if len(args) == 1 {
+		templateFile = args[0]
+	} else if len(args) == 0 {
+		if templateFile = helper.GetDefaultTmplFile(); templateFile == "" {
```

Comment:
If we find more than one template file or no template file, it would be ideal if we returned a detailed error message to the user rather than just dumping them back to the usage information and expecting them to guess what went wrong.

Reply:
Like this?